.circleci/config.yml

```diff
@@ -143,43 +143,76 @@ jobs:
             fi
 
   test-integration:
-    executor: default-machine
+    machine:
+      image: circleci/classic:201808-01
+      docker_layer_caching: true
+    environment:
+      K8S_VERSION: v1.16.0
+      KUBECONFIG: /home/circleci/.kube/config
+      MINIKUBE_VERSION: v1.16.0
+      MINIKUBE_WANTUPDATENOTIFICATION: false
+      MINIKUBE_WANTREPORTERRORPROMPT: false
+      MINIKUBE_HOME: /home/circleci
+      CHANGE_MINIKUBE_NONE_USER: true
+      GOPATH: /home/circleci/go
     steps:
-      - checkout
       - run:
-          name: Set up NVM
+          name: Change the node version
           command: |
-            echo ${NVM_DIR}
             [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-            nvm install v14.17.6
-            nvm alias default v14.17.6
+            nvm install v12.22.1
```

Comment:
Something wrong with the LTS version of node?

Reply:
Not sure why I changed this. Let me try to use the latest version. Thanks for spotting it.